Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -364,6 +355,9 @@ - (UIBarButtonItem*)shareToolbarItem {
 - (void)viewDidLoad {
     [super viewDidLoad];
 
+    [self setupToolbar];
```

Comment:
@coreyfloyd i'm unable to reproduce either, since it now appears that `viewDidLoad` is being called _before_ the `WMFAppViewController` check.  however, it was how I [fixed the regression causing the toolbar to not be displayed](https://github.com/wikimedia/wikipedia-ios/pull/178/files#diff-6797d570310857eec6a38c955002bb24R107). so :+1: i guess?